File: lecture_notes/08 collections/1-array.cpp
#include <stdio.h>

int main() {
  int ages[] = { 10, 12, 15, 15, 17, 18, 18, 19, 20 };

  int first = ages[0];
  int length = sizeof(ages) / sizeof(ages[0]);
  int last = ages[length - 1];


  printf("first: %d\n", first);
  printf("last: %d\n", last);
}

File: lecture_notes/08 collections/1-array.js
'use strict';

const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];

ages.shift();
ages.unshift(5);

ages.pop();
ages.push(5);

const first = ages[0];
const last = ages[ages.length - 1];

console.log({ first, last });
